src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -351,6 +352,18 @@ def add_organization_member(
                 )
         return serialize_member(org_member)
 
+    def update_organization_member(
+        self, *, organization_id: int, member_id: int, attrs: OrganizationMemberUpdateArgs
+    ) -> Optional[RpcOrganizationMember]:
+        member = OrganizationMember.objects.get(id=member_id)
+        with outbox_context(transaction.atomic(router.db_for_write(OrganizationMember))):
+            if len(attrs):
+                for k, v in attrs.items():
+                    setattr(member, k, v)
+                member.save()
```

Comment:
Would it be possible to use the `update` method here instead of manually setting each attribute?

Reply:
I'm not sure how because some of the attributes might not be included, for example if role is not passed I want to keep the role the same as it was before but with an update I have to pass something as role.